hw/ip/kmac/dv/cov/kmac_masked_unr_excl.el

```diff
@@ -1,132 +1,104 @@
 // Copyright lowRISC contributors.
```

Comment:
Auto generated UNR file that black boxed the security common prims.
Also removed the items in `kmac_masked_cov_excl.el` above.